xkcdpassgen.yaml

```diff
@@ -1,15 +1,16 @@
-name: xkcdpassgen
+name: xkcdpass
 version: 1.0
-summary: xkcd password generator snap
+summary: xkcd password generator
+description: Generate secure multiword passwords/passphrases, inspired by XKCD
+
 grade: devel
-confinement: devmode
+confinement: strict
 
 apps:
-    xkcdpass:
-        command: bin/xkcdpass
-parts:
+  xkcdpass:
+    command: bin/xkcdpass
 
-        xkcdpassgen:
-            plugin: python3
-            source: https://github.com/redacted/XKCD-password-generator.git
-            source-type: git
+parts:
+  xkcdpassgen:
+    plugin: python
```

Comment:
python3 is how the plugin used to be called. No that's deprecated and you get the same just using `plugin`.